docs/pages/blog/callback-support-in-style-overrides.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Introducing callback support in style overrides
+description: We're excited to introduce callback support for global theme overrides in this minor version update!
+date: 2022-01-20T00:00:00.000Z
+authors: ['siriwatknp']
+tags: ['MUI Core', 'News']
+card: false
+---
+
+MUI Core [v5.3.0](https://github.com/mui-org/material-ui/releases/tag/v5.3.0) introduces the ability to write a callback in style overrides (global theming), giving you full control of component customization at the theme level.
```

Comment:
This could be clearer:

```suggestion
[MUI Core v5.3.0](https://github.com/mui-org/material-ui/releases/tag/v5.3.0) introduces the ability to write a callback in style overrides (global theming), giving you full control of component customization at the theme level.
```